pkg/api/coinID.go

```diff
@@ -89,12 +89,12 @@ func (c *CoinIDMap) Populate() {
 			symbol := strings.ToUpper(val.Symbol)
 			m.Lock()
 			if _, ok := (*IDMap)[symbol]; ok {
-				(*IDMap)[symbol] = CoinID{
```

Comment:
Type hasn't been changed here too.

Reply:
I've changed the pointer receivers in CurrencyIDMap as well as the places I had missed them. Could you look at it now?

Reply:
finally done